### PR TITLE
[extractor/NaverNow] Fix for site/endpoint changes

### PR DESCRIPTION
**Pull request by ping**
<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### Description of your *pull request* and other information

</details>

This PR fixes the extractor to incorporate site url format and api endpoint changes.

### CODE REVIEW

File: yt_dlp/extractor/naver.py [hunk 1]
```diff
@@ -326,39 +328,41 @@ def _extract_replay(self, show_id, replay_id):
         }, self._extract_video_info(replay_id, vod_info['video_id'], in_key))
 
     def _extract_show_replays(self, show_id):
-        page = 0
+        page_size = 15  # page_size is now a fixed unchangeable value
```
Comment (MinePlayersPE) [inline yt_dlp/extractor/naver.py:331]: imo inlining is better

File: yt_dlp/extractor/naver.py [hunk 2]
```diff
@@ -326,39 +328,41 @@ def _extract_replay(self, show_id, replay_id):
...
                 yield self._extract_replay(show_id, v['id'])
 
-            if try_call(lambda: show_vod_info['count'] <= self._PAGE_SIZE * (page + 1)):
+            if len(show_vod_info.get('vod_list') or []) < page_size:
                 break
             page += 1
 
     def _extract_show_highlights(self, show_id, highlight_id=None):
-        page = 0
+        page_size = 10  # page_size is now a fixed unchangeable value
```
Comment (MinePlayersPE) [inline yt_dlp/extractor/naver.py:347]: ditto

**Review by MinePlayersPE — COMMENTED** [review]
btw are the old replay/highlight IDs completely replaced?

**Comment by pukkandan** [comment]
Is there some way to get the old id?

**Comment by pukkandan** [comment]
> Yes, but why though?

So that download-archives are not broken. But if it needs an additional API call, probably not worth it

File: yt_dlp/extractor/naver.py (hunk 1 above)
Comment (pukkandan) [reply yt_dlp/extractor/naver.py:331]: The whole of _extract_show_replays and _extract_show_highlights needs some refactoring. But beyound the scope of this PR

Comment (pukkandan) [inline yt_dlp/extractor/naver.py:331]:
```suggestion
        page_size = 15
```
I can see it is being set to a constant...

File: yt_dlp/extractor/naver.py (hunk 2 above)
Comment (pukkandan) [inline yt_dlp/extractor/naver.py:347]:
```suggestion
        page_size = 10
```

**Review by pukkandan — CHANGES_REQUESTED** [review]

**Pull request merged**